react-web/src/pages/certification/Certification.tsx

```diff
@@ -252,17 +263,26 @@ const Certification = () => {
               disabled={submitting}
               {...form.register("commit")}
             />
+            <div className="footer">
               <Button
                 type="submit"
-                className="btn btn-primary"
-              buttonLabel={"Start Testing"}
-              showLoader={
-                submitting &&
-                (runStatus !== "finished" && runState !== "failed")
-              }
+                buttonLabel={"Start Testing"}
+                showLoader={
+                  submitting &&
+                  (runStatus !== "finished" && runState !== "failed")
+                }
                 disabled={!form.formState.isValid || submitting}
                 onClick={(_) => setFormSubmitted(true)}
               />
+              {(apiFetching && submitting) && (
+                <Button
```

Comment:
I don't know if adding some extra status condition for this button is the case, but I'm wondering if you have to add a  condition for this abort button's visibility when the certification has already started. 
There is no way to abort/delete a Run after the certification was started. At least not for now.

Reply:
Must be ideal to hold this until ready-for-certification runStatus is integrated. Will revisit this PR

Reply:
Clarification - 'Abort Run' button will be visible only until `status: finished`, i.e. only visible when the test campaign is still running.

I've integrated `aborted` and `ready-for-certification` statuses into History table, also made sure to prevent delete of any campaign from history if the `runStatus: ready-for-certification`.